### ISSUE TITLE
Salesforce Integration - Use Sandbox field with env variable always true

### ISSUE THREAD

**Issue by andrewmenich** [verbb/formie#955]
**Description**
After switching the Use Sandbox field to reference an environment variable, our client was always redirected to their sandbox URL when trying to complete the integration connection. I double-checked that the env variable name was the same in the env file and the project config, that project config had been applied, etc. I also changed the value of the env variable a few times, thinking perhaps I was providing an incorrect value. Values I tried include: 
`SALESFORCE_SANDBOX="false"`
`SALESFORCE_SANDBOX="0"`
`SALESFORCE_SANDBOX=""`
`SALESFORCE_SANDBOX=0`

**Steps to reproduce**
1. Change "Use Sandbox" field value to environment variable in Salesforce Integration settings
2. Click "Connect"

**Additional info**
- Plugin version: 1.6.1
- Craft version: 3.7.45
- Multi-site: Yes or **No**

**Additional context**

Locally I am always redirected to `test.salesforce.com` when "Use Sandbox" is set to an env variable (regardless of what the corresponding env variable is set to.) However, when I explicitly set the project config to `useSandbox: ''`, I am redirect to `login.salesforce.com`.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.2](https://github.com/verbb/formie/releases/tag/1.6.2)
